Joystream/atlas
# POC of captcha

Closes #2969

You can test it under `/playground/captcha`

---
## How can backend verify that the user solved a challenge?
After user solves captcha, it returns secret which we can use to get info about verification
https://docs.hcaptcha.com/#verify-the-user-response-server-side
```BASH
curl https://hcaptcha.com/siteverify \
  -X POST \
  -d "response=CLIENT-RESPONSE&secret=YOUR-SECRET"
```
